sci-libs/vtk/vtk-7.0.0.ebuild

```diff
@@ -0,0 +1,362 @@
+		-DVTK_USE_OGGTHEORA_ENCODER=$(usex theora)
+		-DVTK_USE_NVCONTROL=$(usex video_cards_nvidia)
+		-DModule_vtkFiltersStatisticsGnuR=$(usex R)
+		-DVTK_USE_X=$(usex X)
+	# IO
+		-DVTK_USE_FFMPEG_ENCODER=$(usex ffmpeg)
+		-DModule_vtkIOGDAL=$(usex gdal)
+		-DModule_vtkIOGeoJSON=$(usex json)
+		-DModule_vtkIOXdmf2=$(usex xdmf2)
+	# Apple stuff, does it really work?
+		-DVTK_USE_COCOA=$(usex aqua)
+	)
+
+	if use examples || use test; then
+		mycmakeargs+=( -DBUILD_TESTING=ON )
+	fi
+
+	if use kaapi; then
+		mycmakeargs+=( -DVTK_SMP_IMPLEMENTATION_TYPE="Kaapi" )
+	elif use tbb; then
+		mycmakeargs+=( -DVTK_SMP_IMPLEMENTATION_TYPE="TBB" )
+	else
+		mycmakeargs+=( -DVTK_SMP_IMPLEMENTATION_TYPE="Sequential" )
+	fi
+
+	if use java; then
+		if [[ ${ARCH} == amd64 ]]; then
+			mycmakeargs+=( -DJAVA_AWT_LIBRARY="${JAVA_HOME}/jre/lib/${ARCH}/libjawt.so;${JAVA_HOME}/jre/lib/${ARCH}/xawt/libmawt.so" )
+		else
+			mycmakeargs+=( -DJAVA_AWT_LIBRARY="${JAVA_HOME}/jre/lib/i386/libjawt.so;${JAVA_HOME}/jre/lib/i386/xawt/libmawt.so" )
+		fi
+	fi
+	if use python; then
+		mycmakeargs+=(
+			-DVTK_INSTALL_PYTHON_MODULE_DIR="$(python_get_sitedir)"
+			-DPYTHON_INCLUDE_DIR="$(python_get_includedir)"
+			-DPYTHON_LIBRARY="$(python_get_library_path)"
+			-DSIP_PYQT_DIR="${EPREFIX}/usr/share/sip"
+			-DSIP_INCLUDE_DIR="$(python_get_includedir)"
+			-DVTK_PYTHON_INCLUDE_DIR="$(python_get_includedir)"
+			-DVTK_PYTHON_LIBRARY="$(python_get_library_path)"
+			-DVTK_PYTHON_SETUP_ARGS:STRING="--prefix=${EPREFIX} --root=${D}"
+		)
+	fi
+
+	if use qt5; then
+		mycmakeargs+=(
+			-DVTK_USE_QVTK=ON
+			-DVTK_USE_QVTK_OPENGL=ON
+			-DVTK_USE_QVTK_QTOPENGL=ON
+			-DQT_WRAP_CPP=ON
+			-DQT_WRAP_UI=ON
+			-DVTK_INSTALL_QT_DIR=$(qt5_get_libdir)/qt5/plugins/designer
+			-DDESIRED_QT_VERSION=5
+			-DVTK_QT_VERSION=5
+			-DQT_MOC_EXECUTABLE="$(qt5_get_bindir)/moc"
+			-DQT_UIC_EXECUTABLE="$(qt5_get_bindir)/uic"
+			-DQT_INCLUDE_DIR="${EPREFIX}/usr/include/qt5"
+			-DQT_QMAKE_EXECUTABLE="$(qt5_get_bindir)/qmake"
+			-DVTK_Group_Qt:BOOL=ON
+		)
+	fi
+
+	if use R; then
+		mycmakeargs+=(
+			-DR_LIBRARY_BLAS=/usr/$(get_libdir)/R/lib/libR.so
+			-DR_LIBRARY_LAPACK=/usr/$(get_libdir)/R/lib/libR.so
+		)
+	fi
+
+	append-cppflags -D__STDC_CONSTANT_MACROS -D_UNICODE
+
+	use java && export JAVA_HOME="${EPREFIX}/etc/java-config-2/current-system-vm"
+
+	if use mpi; then
+		export CC=mpicc
+		export CXX=mpicxx
+		export FC=mpif90
+		export F90=mpif90
+		export F77=mpif77
+	fi
+
+	cmake-utils_src_configure
+}
+
+src_test() {
+	local tcllib
+	ln -sf "${BUILD_DIR}"/lib  "${BUILD_DIR}"/lib/Release || die
+	for tcllib in "${BUILD_DIR}"/lib/lib*TCL*so; do
+		ln -sf $(basename "${tcllib}").1 "${tcllib/.so/-${SPV}.so}" || die
+	done
+	export LD_LIBRARY_PATH="${BUILD_DIR}"/lib:"${JAVA_HOME}"/jre/lib/${ARCH}/:"${JAVA_HOME}"/jre/lib/${ARCH}/xawt/
+	virtx cmake-utils_src_test
+}
+
+src_install() {
+	use web && webapp_src_preinst
+
+	cmake-utils_src_install
+
+	if [[ -d ${ED%/}/usr/share/doc/${PN}-${SPV} ]] ; then
```

Comment:
Why do you check for it? When does it (not) exist? Usually you should check for reasons, not effects. I mean, this will silently ignore if documentation dir is not created correctly.